packages/core/aqua-to-js/src/index.ts

```diff
@@ -27,17 +27,14 @@ interface TsOutput {
   sources: string;
 }
 
-type LanguageOutput = {
-  js: JsOutput;
-  ts: TsOutput;
-};
+type LanguageOutput = JsOutput | TsOutput;
```

Comment:
What is `Language`?

Reply:
As type said language is a ts or js. Or you suggest better name?